content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,487 @@
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "saml_assertion_attribute_id": 0
+    },
+    "type": "string",
+    "id": "string",
+    "relationships": {
+      "saml_assertion_attributes": {
+        "data": {
+          "id": 0,
+          "type": "saml_assertion_attributes"
+        }
+      },
+      "roles": {
+        "data": {
+          "id": "string",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "string",
+        "type": "roles",
+        "attributes": {
+          "created_at": "string",
+          "modified_at": "string",
+          "uuid": "00000000-0000-0000-0000-000000000000",
+          "name": "string"
+        },
+        "relationships": {
+          "data": [
+            {
+              "id": "string",
+              "type": "permissions"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "data": {
+        "id": 0,
+        "type": "authn_mappings",
+        "attributes": {
+          "id": 6,
+          "attribute_key": "string",
+          "attribute_value": "string"
+        }
+      }
+    }
+  ]
+}
+{{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Get all AuthN Mappings
+
+Returns a list of AuthN Mappings
+
+| Method | Endpoint path            | Required payload |
+|--------|-------------------------|------------------|
+| `GET`  | `/v2/authn_mappings` | optional queries       |
+
+##### ARGUMENTS
+
+* **`sort`** [*optional*, *default*=**yes**]:
+  Sort attribute and direction - `yes` sorts in descending order, `no` sorts in ascending order.
+* **`page_number`** [*optional*, *default*=**0**, *minimum*=**0**]:
+  The page of results to return. 
+* **`page_size`** [*optional*, *default*=**10**]:
+  The number of results to return on each page.
+* **`filter`** [*optional*, no default]:
+  Filter by tags as strings. For example, `Billing Users`.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X GET "https://app.datadoghq.com/api/v2/authn_mappings" \
+     -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+     -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>"
+```
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholders with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+```json
+{
+  "data": [
+    {
+      "type": "authn_mapping",
+      "id": "string",
+      "relationships": {
+        "saml_assertion_attributes": {
+          "data": {
+            "id": 0,
+            "type": "saml_assertion_attributes"
+          }
+        },
+        "roles": {
+          "data": {
+            "id": "string",
+            "type": "roles"
+          }
+        }
+      },
+      "attributes": {
+        "created_at": "string",
+        "created_by": "string",
+        "uuid": "10000000-abcd-efgh-ijkl-000000000000",
```

Comment:
This uuid actually isn't proper hex, something like `123e4567-e89b-12d3-a456-426655440000` would probably work better.